src/clinical/clinical-service.ts

```diff
@@ -24,6 +24,7 @@ import {
 import { DeepReadonly } from 'deep-freeze';
 import _ from 'lodash';
 import {
+	ClinicalDataSortTypes,
```

Comment:
Due to circular imports I encountered while making this branch, I've created a second PR to move several types into their own file
It's based off of this branch, so should be reviewed after this is merged
https://github.com/icgc-argo/argo-clinical/pull/1185